pex/pex_builder.py

```diff
@@ -521,21 +546,20 @@ def build(self, filename, bytecode_compile=True, deterministic_timestamp=False):
         """
         if not self._frozen:
             self.freeze(bytecode_compile=bytecode_compile)
+        tmp_zip = filename + "~"
         try:
-            os.unlink(filename + "~")
+            os.unlink(tmp_zip)
             self._logger.warning(
-                "Previous binary unexpectedly exists, cleaning: %s" % (filename + "~")
+                "Previous binary unexpectedly exists, cleaning: {}".format(tmp_zip)
             )
```

Comment:
Should this be above the `unlink` call? Wouldn't matter unless it was going to fail, but that's the most useful time to be warned.

Reply:
It would have to be more complex than that since this is in the ask for forgiveness later style. This was existing so I'll leave as-is.